### ISSUE TITLE
Use plain language 

### ISSUE THREAD

**Issue by hiteshah185** [TeraServ/starfish#66]
**Active** and **soft** language should be used for positive user experience. Use words like-Please, Kindly in screens requesting input/feedback from user.
